### DISCUSSION TITLE
Ask if strongswan has been certified.

### DISCUSSION THREAD

**Discussion by gowdamanil** [strongswan/strongswan#2231] [Q&A]
Hello @tobiasbrunner 

Thanks for your previous answers, I was able to set up ipsec based on requirement. But I wanted to know if strongswan is certified by any orgs/authority. This is needed because I have to present it during my code and architecture review.

Thanks 
AGM

**Comment by tobiasbrunner** [comment]
It has been certified by various orgs and authorities in a multitude of products by different vendors to varying degrees (FIPS/CC/BSI etc.). Since 5.9.6, strongSwan can delegate all crypto operations (including the IKEv2 key derivation) to a FIPS-certified crypto library (e.g. OpenSSL or wolfSSL), which should not require any specific crypto certification of strongSwan itself. Other certifications may require more work.

**Reply by gowdamanil** [reply]
Thanks tobias, this will be helpful.
